.github/ISSUE_TEMPLATE/new_component.md

```diff
@@ -39,25 +38,46 @@ about: Interested in contributing a new component to Fluent UI React-Native? Thi
 - [ ] macOS
 - [ ] win32
 - [ ] windows
+- [ ] android
 
 ## Component code
 
 ### Example code
 
-```Swift
+```javascript
 ```
 
 ### New Component Checklist
 
-- [ ] Light and Dark Appearance
+General:
 - [ ] Unit tests
 - [ ] UI tests
 - [ ] Different resolutions (1x, 2x, 3x)
```

Comment:
These are all still Apple specific (terminology of UI Tests vs. Unit Tests) and possibly not relevant to our testing plan here. If we split tests into specific types, we should use the exact terminology of the type of tests (i.e. Appium). For now just "Automated Tests" could be relevant to all platforms? And the UI resolutions of 1x,2x,3x is strictly an Apple thing

Reply:
Thanks, I definitely don't know all the cross-plat terminology. Automated Tests works. I guess other platforms aren't concerned about multiple resolutions because they use SVGs?